Algorithms/Python/Sorting/bubble_sort.py

```diff
@@ -0,0 +1,38 @@
+"""
+    Bubble sort is a sorting algorithm in which every element is compared and swapped so that the last index is filled
+    with the correct element first
+"""
+
+
+def bubble_sort(arr, order=None):
```

Comment:
introduce an optimization to stop sorting if no swaps are made in a pass, as the array is already sorted